File: sdks/thunkmetrc-go/wrapper/models/job_types_category.go
package models

type JobTypesCategory struct {
    ForItems bool `json:"ForItems,omitempty"`
    ForProcessingJobs bool `json:"ForProcessingJobs,omitempty"`
    Id int64 `json:"Id,omitempty"`
    Name string `json:"Name,omitempty"`
    RequiresProcessingJobAttributes bool `json:"RequiresProcessingJobAttributes,omitempty"`
}

File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/JobTypesCategory.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class JobTypesCategory {
    @JsonProperty("ForItems")
    public Boolean forItems;
    @JsonProperty("ForProcessingJobs")
    public Boolean forProcessingJobs;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("Name")
    public String name;
    @JsonProperty("RequiresProcessingJobAttributes")
    public Boolean requiresProcessingJobAttributes;
}
